FlowCryptAppTests/Core/FlowCryptCoreTests.swift

```diff
@@ -387,9 +388,6 @@ final class FlowCryptCoreTests: XCTestCase {
         }
     }
 
-    // this test is only meaningful on a real device
```

Comment:
Now it makes sense and also helped catch bug

Reply:
Let's add `This test always passes, even wrongly, on simulators running on a mac with 2 or fewer cores. Behaves meaningfully on real iPhone or simulator on a mac with many cores`